File: C++11/백준/Bronze/16693. Pizza Deal/Pizza Deal.cc
#include <iostream>
#include <math.h>
using namespace std;
int main(){
    double a[2];
    double b[2];
    cin >> a[0] >> a[1] >> b[0] >> b[1];
    double c1 = a[0]/a[1];
    double c2 = (b[0]*b[0]*M_PI)/b[1];
    if(c1>c2){
        cout << "Slice of pizza";
    }
    else{
        cout << "Whole pizza";
    }
}

File: Java/백준/Bronze/16693. Pizza Deal/Pizza Deal.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        double[] a = new double[4];
        for (int i=0; i<4; i++){
            a[i] = in.nextDouble();
        }
        double c1 = a[0]/a[1];
        double c2 = (a[2]*a[2]*Math.PI)/a[3];
        if(c1>c2){
            System.out.println("Slice of pizza");
        }
        else{
            System.out.println("Whole pizza");
        }
    }
}